src/Backups/BackupIO_Disk.h

```diff
@@ -34,7 +35,11 @@ class BackupWriterDisk : public IBackupWriter
     bool fileContentsEqual(const String & file_name, const String & expected_file_contents) override;
     std::unique_ptr<WriteBuffer> writeFile(const String & file_name) override;
     void removeFiles(const Strings & file_names) override;
+    DataSourceDescription getDataSourceDescription() const override;
 
+    bool supportNativeCopy(DataSourceDescription data_source_description) const override;
```

Comment:
Why not pass `DataSourceDescription` by const reference? Even though this struct is small

Reply:
Because sometimes it's not suitable